tests/migration/online_migration/pre_migration.pm

```diff
@@ -19,15 +19,17 @@ use x11utils 'turn_off_gnome_show_banner';
 
 sub check_or_install_packages {
     if (get_var("FULL_UPDATE") || get_var("MINIMAL_UPDATE")) {
-        # if system is fully updated or even minimal patch applied, all necessary packages for online migration should be installed
-        # check if the packages was installed along with update
-        my $output = script_output "rpm -qa yast2-migration zypper-migration-plugin rollback-helper | sort";
-        if ($output !~ /rollback-helper.*?yast2-migration.*?zypper-migration-plugin/s) {
-            record_soft_failure 'bsc#982150: migration packages were not installed along with system update. Installing missed package to continue the test';
-            zypper_call "in yast2-migration zypper-migration-plugin rollback-helper snapper";
+        if (is_leap_migration) {
+            zypper_call('rm zypper-migration-plugin');
```

Comment:
Is there any reason why need this step? Any document mention this? If yes, we need mention in code.

Reply:
https://bugzilla.suse.com/show_bug.cgi?id=1197268#c2
Comment # 2 on bug 1197268 from Jacek Tomasiak
just like with https://bugzilla.suse.com/show_bug.cgi?id=1196571 the solution
is to remove old package (zypper-migration-plugin) from Leap 15.4.

Reply:
added comment there.